api/src/main/java/org/openmrs/module/eptsreports/reporting/library/queries/TXTBQueries.java

```diff
@@ -540,7 +540,7 @@ public static String getPatientsWithObsBetweenDates(
     s.append("WHERE e.location_id = :location AND e.encounter_type = ${encounterType} ");
     s.append("AND o.concept_id = ${question}  ");
     s.append("AND o.value_coded in (${answers}) ");
```

Comment:
Same here

```suggestion
    s.append("AND o.concept_id = ${malariaConcept}  ");
    s.append("AND o.value_coded in (${aspirinaConcept}) ");
```

Reply:
I guess this is like this cause is generic one, never mind the sugestion.....
Please move this to genericQueries